terraform/lb.tf

```diff
@@ -0,0 +1,44 @@
+# https://github.com/hashicorp/terraform-provider-google-beta/pull/2348
+# https://cloud.google.com/load-balancing/docs/negs/setting-up-serverless-negs
+resource "google_compute_region_network_endpoint_group" "apiserver-us-central1" {
+  provider  = google-beta
+  region    = "us-central1"
+  cloud_run = "apiserver"
+}
+
+resource "google_compute_backend_service" "apiserver" {
```

Comment:
I'm a bit concerned about how this will impact the x-forwarded-for and remote-ip headers in our rate limiter.

Reply:
yeah, I'm going to send a PR for that, we're basically getting lucky right now. In theory this header should be an array of IPs.